### PR TITLE
Re-enable end_tx tests that depended on SSTORE

### PR DESCRIPTION
**Pull request by rrtoledo**
### Description

Enable tests to check if gas refund was implemented correctly (c.f. [EIP 3529](https://eips.ethereum.org/EIPS/eip-3529))

### Issue Link

Issue #1111 

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Contents

- Uncommenting and updating tests

### Rationale

Using the commented values and common values for account creation.

### CODE REVIEW

File: zkevm-circuits/src/evm_circuit/execution/end_tx.rs [hunk 1]
```diff
@@ -325,19 +327,44 @@ mod test {
...
+                            .storage(vec![(key, original_value)].into_iter());
+                        accs[1]
+                            .address(MOCK_ACCOUNTS[1])
+                            .balance(Word::from(10u64.pow(19)));
+                    },
+                    |mut txs, accs| {
+                        txs[0]
+                            .to(accs[0].address)
+                            .from(accs[1].address)
+                            .value(tx_value)
```
[the remark this answers is not in the document]
Comment (rrtoledo) [reply zkevm-circuits/src/evm_circuit/execution/end_tx.rs]: Just saw this message now. That was very useful and I should have had a look at it earlier!
This being said, I just pushed a commit fixing the tests.

**Comment by rrtoledo** [comment]
> LGTM! Just printed the trace locally and the comments look correct, it's pretty clear now! Nice work!

Thanks! Do you think I should add some commented code to easily check the balances and the steps' refunds?

**Comment by han0110** [comment]
> Thanks! Do you think I should add some commented code to easily check the balances and the steps' refunds?

Actually I used `revm` to reproduce the result, but I'm also curious about what's your trick/code to check the refund and balance, so it'd be cool to also have the instructions to show that, but your choice!

**Review by ed255 — APPROVED** [review]
LGTM!  It's great to see the thorough explanation of the tests and the instructions to observe the expected behavior!  Nice work!

**Pull request merged**